lizmap/www/js/attributeTable.js

```diff
@@ -1297,7 +1297,7 @@ var lizAttributeTable = function() {
                     // Check if we need to replace url or media by link
                     // Add function for any string cell
                     // First check if the col is number
-                    if (idx in cTypes && cTypes[idx] == 'integer')
+                    if (idx in cTypes && (cTypes[idx] == 'integer' || cTypes[idx] == 'int'))
```

Comment:
Can you rebuild this by using switch. It will be easiest to cover all the numbers description.

Reply:
Do we want 'int' and 'integer' types ? https://github.com/qgis/QGIS/blob/release-2_18/src/server/qgswfsprojectparser.cpp#L473

Reply:
Yes, the twice are possible and all other integer types.